src/Forms/Main.cs

```diff
@@ -20333,6 +20337,12 @@ private void NormalToolStripMenuItem1Click(object sender, EventArgs e)
         {
             var tb = GetFocusedTextBox();
 
+            // minimum length of valid html tag is 3
+            if (tb.SelectionLength < 3)
+            {
+                return;
+            }
+
             if (tb.SelectionLength == 0)
             {

```

Comment:
This condition (`tb.SelectionLength == 0`) can no longer be true?

Reply:
> This condition (`tb.SelectionLength == 0`) can no longer be true?

Moved above, thanks!